src/modules/uORB/Subscription.cpp

```diff
@@ -64,12 +64,6 @@ bool Subscription::subscribe()
 			if (node != nullptr) {
 				_node = node;
 				_node->add_internal_subscriber();
-
-				const unsigned curr_gen = _node->published_message_count();
-
-				// If there were any previous publications allow the subscriber to read them
-				_last_generation = curr_gen - math::min((unsigned)_node->get_queue_size(), curr_gen);
```

Comment:
These codes can be deleted, _last_generation is 0 by default and there is no problem, uORB::DeviceNode will automatically set it to a valid value.

Reply:
Please check https://github.com/PX4/Firmware/pull/15709#issuecomment-699513038
This part of the code will be modified again soon.